ddtrace/contrib/async_trace_utils.py

```diff
@@ -0,0 +1,34 @@
+from ddtrace import Pin
+from ddtrace.internal.logger import get_logger
+
+
+log = get_logger(__name__)
+
+
+def with_traced_module(func):
```

Comment:
Can we move this to `trace_utils` and suffix it with `_async`? Unless you think this set of utility function for async would grow and be better off maintained separately.

Reply:
We can't keep it in normal trace_utils as there would be compatibility issues with 2.7 which is why it's in its own module.

I'll add a docstring to the module to point this out though 👍